flash/1.2.11/tests/scripts/run_flash_pos_control.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# This script runs FLASH to merge overlapping paired-end reads.
+
+R1=/data/R1.fastq.gz
+R2=/data/R2.fastq.gz
+OUTDIR=/test_result
+mkdir -p $OUTDIR
+
+# Get read length, make minimum overlap 80% of read length
+READ_LEN=$(gunzip -c "$R1" | head -n 400 \
```

Comment:
Since you are using test data with a known read length, is this really required?

Reply:
Nope, not required. I'll remove it.